src/LinkedList.java

```diff
@@ -31,6 +30,17 @@ public void addElement(Node node) {
         lengthOfList++;
```

Comment:
rename `lengthOfList `to `length`, because this variable is the attribute of the LinkedList class, which makes us clear, that this is the length of the List